sdk/schemaregistry/schema-registry/README.md

```diff
@@ -64,8 +58,9 @@ schema registry.
 
 ### SchemaRegistry serializers
 
-- [SchemaRegistryAvroSerializer](../schema-registry-avro) is a separate package that
-  uses `SchemaRegistryClient` to pair schema ID along with Avro Binary Encoding.
+- [SchemaRegistryAvroSerializer](https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/schemaregistry/schema-registry-avro)
```

Comment:
Use package name since we are referring to "separate package" in the following line

```suggestion
- [@azure/schema-registry-avro](https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/schemaregistry/schema-registry-avro)
```